packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -398,6 +397,18 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     });
     return response.map(deserializeExecutionPayloadBody);
   }
+
+  async isOffline(): Promise<boolean> {
+    try {
+      await this.rpc.fetch({method: "eth_chainId", params: []}, {});
```

Comment:
how should timeouts be handled? I feel waiting 12 seconds here for a response is not great. 

AFAIK only Lighthouse VC uses el_offline at the moment, we might wanna check how their beacon node handles this.